mkosi/__init__.py

```diff
@@ -1792,13 +1792,14 @@ def setup_dnf(args: CommandLineArguments, root: str, repos: List[Repo] = []) ->
             )
 
     config_file = os.path.join(workspace(root), "dnf.conf")
+    default_repos = f"{'repodir' if args.distribution == Distribution.photon else 'reposdir'}={workspace(root)}"
```

Comment:
Maybe write this as
```python
if args.use_system_repositories:
   default_repos = ''
else:
   default_repos = f"{'repodir' if args.distribution == Distribution.photon else 'reposdir'}={workspace(root)}\n"
```
This will be longer but clearer, I think.

Reply:
Applied suggestion.

Rebased and added the additional details into the commit message.